### PR TITLE
feat: add localisation for maps

### PR DESCRIPTION
**Pull request by gonzalezad1**

### CODE REVIEW

File: db/queries/tis.js [hunk 1]
```diff
@@ -35,10 +35,35 @@ const getTiByUserId = userId =>
...
+      "tis.id",
+      "geolocalisation_code_postal.latitude",
+      "geolocalisation_code_postal.longitude",
+      "users_tis.cabinet",
+      "users_tis.email",
+      "users.type",
+      "users.last_login",
+      "users.active",
+      "users.reset_password_token",
+      "users.reset_password_expires"
```
Comment (revolunet) [inline db/queries/tis.js]: rm champs inutiles

File: routes/usersTi.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  getTiByUserIdWithCodePostal(req.user.id)
+    .then(function(ti) {
+      res.status(200).json(ti);
+    })
+    .catch(function(error) {
+      throw error;
+      next(error);
+    });
+});
+
```
Comment (revolunet) [inline routes/usersTi.js]: add a single test

File: routes/usersTi.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+var express = require("express");
+var router = express.Router();
+
+const { loginRequired, typeRequired } = require("../auth/_helpers");
+const { getTiByUserIdWithCodePostal } = require("../db/queries/tis");
+
+// TODO:  Security for mandataire
```
Comment (douglasduteil) [inline routes/usersTi.js]: Put your name on it 🙂

File: routes/usersTi.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+ *       - tis
+ *     description: get ti information
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: Object
```
Comment (douglasduteil) [inline routes/usersTi.js]: Ti object aren't defined ?
Missing Todo ?

File: routes/usersTi.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+router.get("/", loginRequired, typeRequired("ti"), async (req, res, next) => {
+  if (req.user.type !== "ti") {
+    return next(new Error(401));
+  }
+  getTiByUserIdWithCodePostal(req.user.id)
+    .then(function(ti) {
+      res.status(200).json(ti);
+    })
+    .catch(function(error) {
+      throw error;
```
Comment (douglasduteil) [inline routes/usersTi.js]: If you throw here the "next(error)" won't run

**Pull request merged**